src/renderers/webxr/WebXRController.js

```diff
@@ -139,55 +123,64 @@ Object.assign( WebXRController.prototype, {
 
 				handPose = true;
 
-				for ( let i = 0; i <= window.XRHand.LITTLE_PHALANX_TIP; i ++ ) {
+				for ( const inputjoint of inputSource.hand.values() ) {
```

Comment:
Is it possible to turn this into a simple `for` loop instead of a `for of` loop?

Reply:
What would be the advantage of doing it that way?

Reply:
Avoiding having to add a polyfill for it:
https://github.com/mrdoob/three.js/commit/657b36d6132a9f522b11f6277e438d7171528a03#diff-af408426d6a7b072569230b0ee45fa87b6e69f3e97ebdc527d6995924aa1a1ceR1061-R1102